runtime/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp

```diff
@@ -0,0 +1,540 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "pyopenvino/graph/preprocess/pre_post_process.hpp"
+
+#include <pybind11/functional.h>
+#include <pybind11/stl.h>
+
+#include "openvino/core/function.hpp"
+#include "openvino/core/node.hpp"
+#include "openvino/core/preprocess/pre_post_process.hpp"
+
+namespace py = pybind11;
+
+static void regclass_graph_PreProcessSteps(py::module m) {
+    py::class_<ov::preprocess::PreProcessSteps, std::shared_ptr<ov::preprocess::PreProcessSteps>> steps(
```

Comment:
To implement 'getters preprocessing API', you can use custom holder type holding returned reference

So that to allow users to write code like `preprocessor.input(0).preprocess().mean(127).scale(127)` you can define custom holder for PreProcessSteps and InputInfo classes.
See example here:
https://github.com/openvinotoolkit/openvino/pull/8751/commits/fd794dc458db565e5917e2fc1801fe848d7e1ee1